ssn/ssn_separated/ssn.ttl

```diff
@@ -15,16 +15,16 @@
 @base <http://www.w3.org/ns/ssn/> .
 
 <http://www.w3.org/ns/ssn/> rdf:type owl:Ontology , voaf:Vocabulary ;
-                             dc:creator "W3C Spatial Data on the Web Working Group“^^xsd:string ;
-                             dct:modified "2017-04-17” ;
+                             dc:creator "W3C/OGC Spatial Data on the Web Working Group"^^xsd:string ;
+                             dct:modified "2017-04-17" ;
                              owl:versionInfo "New modular version of the SSN ontology."^^xsd:string ;
-                             rdfs:comment "Please report any errors to the W3C Spatial Data on the Web Working Group via the SDW WG Public List public-sdw-wg@w3.org"^^xsd:string ;
+                             skos:note "Please report any errors to the W3C Spatial Data on the Web Working Group via the SDW WG Public List public-sdw-wg@w3.org"^^xsd:string ;
                              rdfs:seeAlso "https://www.w3.org/2015/spatial/wiki/Semantic_Sensor_Network_Ontology" ;
                              dc:title "Semantic Sensor Network Ontology" ;
                              dct:modified "2017-04-17" ;
                              dc:identifier "http://www.w3.org/ns/ssn" ;
                              dct:license <http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document> ;
-                             rdfs:comment "This ontology is developed by the W3C Semantic Sensor Networks Incubator Group (SSN-XG). The concepts and structure of the ontology were discussed in the group's meetings and on the mailing list. For more information on the group's activities see: http://www.w3.org/2005/Incubator/ssn/"^^xsd:string ;
+                             skos:historyNote "This ontology was originally developed in 2009-2011 by the W3C Semantic Sensor Networks Incubator Group (SSN-XG). For more information on the group's activities see: http://www.w3.org/2005/Incubator/ssn/. The ontology was revised and modularized in 2015-2017 by the W3C/OGC Spatial Data on the Web Working Group, see: https://www.w3.org/2015/spatial/wiki/Semantic_Sensor_Network_Ontology. In particular, (a) the scope is extended to include actuation and sampling; (b) the core concepts and properties are factored out into the SOSA ontology. The SSN ontology imports SOSA and adds formal axiomatization consistent with the text definitions in SOSA, and adds classes and properties to accommodate the scope of the original SSN ontology. "^^xsd:string ;
```

Comment:
I think we said -> skos:historyNote are for terms, whereas owl:versionInfo are for ontologies.
See resolution 2017-02-28 described in https://www.w3.org/2015/spatial/wiki/Ontology_versionInfo